dracut/module-setup.sh

```diff
@@ -14,6 +14,12 @@ depends() {
     return 0
 }
 
+installkernel() {
+    case "$(uname -m)" in
+        s390*) instmods hmcdrv ;;
+    esac
```

Comment:
Why to use `case` when it is only one value tested here?

Reply:
It is the same as in the function `depends` above. I think that the point is that we can more easily extend it with other cases.

Reply:
Ok.